crates/ruff_db/src/files.rs

```diff
@@ -145,6 +178,14 @@ impl Files {
                 file.sync(db);
             }
         }
+
+        let routes = inner.roots.read().unwrap();
```

Comment:
I don't think the word "routes" makes sense here?
```suggestion
        let roots = inner.roots.read().unwrap();
```